sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImplBuilder.java

```diff
@@ -181,7 +197,7 @@ public TextAnalyticsClientImpl buildClient() {
         if (serializerAdapter == null) {
             this.serializerAdapter = JacksonAdapter.createDefaultSerializerAdapter();
         }
-        TextAnalyticsClientImpl client = new TextAnalyticsClientImpl(pipeline, serializerAdapter, endpoint);
+        TextAnalyticsClientImpl client = new TextAnalyticsClientImpl(pipeline, serializerAdapter, apiVersion, endpoint);
```

Comment:
There should be a default value for ApiVersion here in case this is used directly

Reply:
it is the autogenerated class. The default value is assigned In the convenience level. https://github.com/Azure/azure-sdk-for-java/pull/21028/files?file-filters%5B%5D=.java&hide-deleted-files=true#diff-58e093029f6be3020e93b0556d2f991c51174067b3ceba5d59744230f6a5824aL148